packages/opencensus-core/src/trace/model/types.ts

```diff
@@ -406,7 +406,7 @@ export interface Span {
    * @param key Describes the value added.
    * @param value The result of an operation.
    */
-  addAttribute(key: string, value: string|number|boolean): void;
+  addAttribute(key: string, value: string|number|boolean|object): void;
```

Comment:
Document that `object` must be JSON.stringify-able or else it will raise an exception.

Reply:
Documentation added

Reply:
Could you please update signature in `NoRecordSpan` class? https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/model/no-record/no-record-span.ts#L152